core/src/main/java/com/novoda/merlin/MerlinsBeard.java

```diff
@@ -69,4 +69,12 @@ public boolean isConnectedToMobileNetwork() {
         return networkInfo != null && networkInfo.isConnected();
     }
 
+    public String getMobileNetworkSubtypeName() {
+        NetworkInfo networkInfo = getNetworkInfo();
+        if (networkInfo == null || !networkInfo.isConnected()) {
+            return "";
```

Comment:
:+1: ``return "unknown";` is also good for me. But not sure that's what he meant :tongue: